src/Mocks/Editor/TextSelectionMock.cs

```diff
@@ -9,46 +9,36 @@
 using Microsoft.VisualStudio.Text.Editor;
 using Microsoft.VisualStudio.Text.Formatting;
 
-namespace Microsoft.VisualStudio.Editor.Mocks
-{
+namespace Microsoft.VisualStudio.Editor.Mocks {
     [ExcludeFromCodeCoverage]
-    public sealed class TextSelectionMock : ITextSelection
-    {
+    public sealed class TextSelectionMock : ITextSelection {
         private ITextRange _range;
 
         public TextSelectionMock(ITextView textView, int position) :
-            this(textView, new TextRange(position, 0))
-        {
+            this(textView, new TextRange(position, 0)) {
         }
 
-        public TextSelectionMock(ITextView textView, ITextRange range)
-        {
+        public TextSelectionMock(ITextView textView, ITextRange range) {
             TextView = textView;
             _range = range;
         }
 
         public bool ActivationTracksFocus { get; set; }
 
-        public VirtualSnapshotPoint ActivePoint
-        {
-            get
-            {
+        public VirtualSnapshotPoint ActivePoint {
```

Comment:
If you're changing formatting here, you can reduce it to `public VirtualSnapshotPoint ActivePoint => AnchorPoint;`